### ISSUE TITLE
*.md rendering

### ISSUE THREAD

**Issue by armtir** [xinyi-joffre/azure-devops-mermaid-renderer#1]
Hi, 
Thanks for this very useful extension. Is rendering of *.md files also on your task list? It would be nice to visualize mermaid diagrams within README.md files too.

**Comment by jrobin5** [comment]
I agree as it appears the mmd files cannot accessed from within md files.

**Comment by snewz** [comment]
I'd like to use mermaid diagramms embedded in markdown too.

**Comment by oceanebrit** [comment]
This feature is really important - most of us use readme.md file.
